scripts/finance/expenses-json.js

```diff
@@ -0,0 +1,36 @@
+const fs = require('fs');
+const yaml = require('yaml');
+const path = require('path');
+const { convertToJson } = require('../utils');
+
+// Function to read YAML content from a file
+function readYamlFromFile(filePath) {
+    try {
+        const yamlContent = fs.readFileSync(filePath, 'utf8');
+        const parsedYaml = yaml.parse(yamlContent); // Parse YAML using the "yaml" package
+        return parsedYaml;
+    } catch (error) {
+        console.error('Error reading YAML file:', error.message);
+        return null;
+    }
+}
```

Comment:
I would prefer you extend `convertToJson` with such validation, instead of having it in a custom script. It is a generic functionality that should be applied for any script that uses `convertToJson`

Reply:
But @vishvamsinh28 do make sure, you don't break any functionality inside other scripts as well.